site/assets/scss/_navbar.scss

```diff
@@ -1,6 +1,7 @@
 .bd-navbar {
   padding: .75rem 0;
   background-color: transparent;
+  forced-color-adjust: none;
```

Comment:
Setting  `forced-color-adjust: none` might not go well with "user choices to be respected".

See https://developer.mozilla.org/en-US/docs/Web/CSS/forced-color-adjust#usage_notes

> This property should only be used to makes changes that will support a user's color and contrast requirements. For example, if you become aware that the color optimizations made by the user agent result in a poor experience when in a high contrast or dark mode. Using this property would enable tweaking of the result in that mode to provide a better experience. It should not be used to prevent user choices being respected.

Reply:
Agreed. I think the PR should have this:

```
@media (forced-colors) {
    background-color: Canvas;
  }
```  
(I did a lot of research and testing on this for: https://github.com/coliff/bootstrap-forced-colors-css#readme)

Reply:
Hi @coliff , @rr-it . Thanks for the suggestion! I’ve made the changes. Thank you!